resources/views/layout/main.php

```diff
@@ -29,101 +29,81 @@
 $this->beginPage()
 ?><!DOCTYPE html>
 <html lang="<?= Html::encode($locale->language()) ?>">
+
```

Comment:
Why?

Reply:
What?
To be
     "$this->beginPage()
     ?><!DOCTYPE html>"
and not
     "$this->beginPage()
     ?>
     <!DOCTYPE html>"
?

https://t.me/yii3en/9570

Should I change?